src/configMaker.js

```diff
@@ -1,4 +1,4 @@
-import _ from "lodash";
+import { get, merge, set } from "lodash";
```

Comment:
prefer importing them one at a time
```js
import get from "lodash/get";
(...)
```

Reply:
Why?

Reply:
AFAIK, we completely removed these from phoenix, and want to remove it from styx as well, something to do with memory and performance (because lodash is so large, doing the deconstructive import takes all of lodash into memory, and then takes what you ask)

it's not that important 😄

Reply:
The fact that we removed this from phoenix is not a reason. It's just something we did and might not have had a good reason. Or it might have had a reason that is only relevant on FE because we want to reduce the in-browser footprint.

I vaguely recall from those phoenix discussions that using `lodash/get` didn't actually reduce memory usage. If you want to continue this argument please be prepare to present numbers.

Reply:
let's put it this way - it does make sense that it takes a bit more memory, but when this is the thing we're left with optimizing, we're in a great place 😆